.github/autobuild/android.sh

```diff
@@ -100,10 +100,10 @@ build_app_as_apk() {
 }
 
 pass_artifact_to_job() {
-    mkdir deploy
+    mkdir publish
     local artifact="jamulus_${JAMULUS_BUILD_VERSION}_android.apk"
```

Comment:
uploading everything from the deploy directory doesn’t work as
1. GH on Windows doesn‘t support wildcards
2. MacOS moves the binary (.app files) to the deploy folder

If instead we upload the files by name - as we do in this state - we could also use the deploy directory directly.